build/ci/github_actions/wait_for_k8s_ready.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash -xe
+set +o pipefail
+
+are_pods_ready (){
+  pods=$@
+  are_all_pods_ready=false
+  for pod in $pods; do
+    running_containers_count=`echo $pod | awk -F / '{print$1}'`
+    total_containers_count=`echo $pod | awk -F / '{print$2}'`
+    if [ $running_containers_count != $total_containers_count ]; then
+      are_all_pods_ready=true
+      break
+    fi
+  done
+  echo $are_all_pods_ready
+}
+
+is_kubernetes_cluster_ready (){
+  pods=`kubectl get pods -A | awk '{print$3}' | grep -iv ready`
+  all_containers_are_running=false
+  are_all_pods_ready=$(are_pods_ready $pods)
+  if [ $are_all_pods_ready == "false" ]; then
+    all_containers_are_running=true
+  fi
+  
+  echo $all_containers_are_running
```

Comment:
is `all_containers_are_running` really necessary?
and if so, please align it to the boolean naming convention here, which is to start with `are`

Reply:
yes they are important
done

Reply:
yes they are important
done

Reply:
thanks
so why do we need them (and not just echo true/false)?

Reply:
> thanks (though I still see `all_containers_are_running`).
>
I don't see any `all_containers_are_running`

Reply:
> so why do we need them (and not just echo true/false)?
>
done